api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -9359,3 +9362,96 @@ message AppAuthConfigVerify {
     (gogoproto.jsontag) = ""
   ];
 }
+
+// VnetConfigCreate is emitted when a VnetConfig is created.
+message VnetConfigCreate {
+  // Metadata is a common event metadata.
+  Metadata metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // Resource is a common resource event metadata.
+  ResourceMetadata resource = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
```

Comment:
not sure we need this since it's a singleton resource and all you put in here is the name, could treat it the same as cluster_networking_config